src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVideo.jsx

```diff
@@ -17,26 +19,16 @@ import VideoInstructionsLink from './VideoInstructionsLink';
 import VideoLocation from './VideoLocation';
 
 function formatHeader(appointment) {
-  const patientHasMobileGfe =
```

Comment:
I simplified this by using existing functions in the appointment service and also added the `isGfeVideoAppointment` function. I have some ideas about how to further clean up some of code throughout the project that susses out the type of appointment. I'll address that in another PR.